src/components/application_manager/src/commands/mobile/register_app_interface_request.cc

```diff
@@ -805,9 +812,21 @@ void RegisterAppInterfaceRequest::SendRegisterAppInterfaceResponseToMobile() {
     resumption = resumer.IsApplicationSaved(application->policy_app_id(),
                                             application->mac_address());
   }
+  AppHmiTypes hmi_types;
+  if ((*message_)[strings::msg_params].keyExists(strings::app_hmi_type)) {
+    smart_objects::SmartArray* hmi_types_ptr =
```

Comment:
@okozlovlux I think that if app_hmi_type is not array, this will be detected on schema JSON validation and INVALID_DATA will be returned. Am I wrong?